tests/integ/sagemaker/jumpstart/model/test_jumpstart_model.py

```diff
@@ -29,6 +33,8 @@
 
 MAX_INIT_TIME_SECONDS = 5
 
+MODEL_PACKAGE_ARN_SUPPORTED_REGIONS = {"us-west-2", "us-east-1"}
```

Comment:
FYI: this is going to expand very soon to 16 or so regions

Reply:
Yes I know. If I don't do this then the test would fail in the other regions.

Reply:
just FYI, we might only need `IAD`, PDX`, `DUB` and `SIN` initially, and see if we receive PFRs